src/shapes/text.class.js

```diff
@@ -148,7 +148,8 @@
     linethrough:       false,
 
     /**
-     * Text alignment. Possible values: "left", "center", "right" or "justify".
+     * Text alignment. Possible values: "left", "center", "right", "justify",
+     * "justify-center" or "justify-right".
```

Comment:
and justify-left too.

Reply:
I now realise what you meant here. I thought we were going to overwrite the behaviour of "justify", but we are keeping that one intact and are adding "justify-left". Are you going to change this in https://github.com/kangax/fabric.js/pull/4437 or should i make another PR?